### ISSUE TITLE
tails-amd64-4.13.iso won't work in UEFI Mode!

### ISSUE THREAD

**Issue by ABOhiccups** [ventoy/Ventoy#626]
Tails Version 4.13 does works perfect in Legacy BIOS Mode.

But it won't work in UEFI Mode.

![IMG-2586](https://user-images.githubusercontent.com/17691106/100943749-d1ee6880-34cb-11eb-92f4-bd53c7b080db.jpg)

**Comment by ventoy** [comment]
I can't reproduce the problem.
I test it and it works fine in my laptop (both Legacy BIOS and UEFI mode).
```
tails-4.13-amd64.iso
md5sum: 1121970d5891d89b4a1b05bccc3ac9d4
```

Does other ISO files work fine in the same envrioment? (e.g. Ubuntu)

**Comment by ABOhiccups** [comment]
I tested on Desktop PC. Tails does work in UEFI Mode through Ventoy.

The one I tested was **Dell Inspiron 3520** Laptop and it won't work with Tails in UEFI Mode through Ventoy. That's weird!

**Comment by ventoy** [comment]
Does other ISO files work fine in the  Dell Inspiron 3520  in UEFI mode? (e.g. Ubuntu/CentOS)

**Comment by ABOhiccups** [comment]
Yes. I tested Boot-Repair-Disk, MemTest86, Parted Magic, Kaspersky Rescue Disk, Arch Linux, and Windows 10.

Both ISOs (except Tails) works perfect on Dell Inspiron 3520 in UEFI Mode.

**Comment by ventoy** [comment]
That's really weird!

**Comment by ABOhiccups** [comment]
Taile 4.12 was tested. Same issue happened.

**Comment by ventoy** [comment]
What about tails-amd64-4.13.img (USB image file) ?

**Comment by ABOhiccups** [comment]
It boots to Grub screen perfect. But it stuck at Plymouth Splash Screen.

**Comment by ventoy** [comment]
This is expected. But at least it boot to grub screen, no similar error happened.
I have no idea about this currently.

**Comment by ventoy** [comment]
I think I found the problem.

You can do the following test.

Use a VirtualMachine(e.g. VMware/VirtualBox) and set to UEFI mode.
Directly connect the `tails-4.13-amd64.iso` to the VM.
Then you will find that it can NOT boot in UEFI mode.

That is to say, the tails ISO file does not support UEFI boot itself (in ISO mode).
If you extract the whole ISO to a USB drive with fat32 partition, it can boot in UEFI mode but this is not ISO mode.
The developer of tails miss some option to make it support UEFI mode when generating the ISO file.
(You can test ubuntu.iso in the same way, you will find it can boot normally in UEFI mode).

Ventoy use a special way to do its best to boot these similar ISO files in UEFI mode, but this special method has some compatibility issues with some UEFI firmware.
That's why it works in my laptop and your PC, but has problem with your Dell Inspiron 3520.

So the most fundamental reason is that tails ISO file does NOT support UEFI mode itself.
I hope the developer of tails can add support in future releases.

**Comment by ventoy** [comment]
What about this release:  https://github.com/ventoy/Ventoy/actions/runs/613266157
